backend/generator.js

```diff
@@ -37,7 +37,7 @@ exports.executeScript = function (socket, formData, callback) {
     "-d", debug,
     "-u", uniqueId,
     "-s", subProject,
-    "-p", subDocpath.join(","),
+    "-p", "[" + subDocpath.join(",") + "]",
```

Comment:
Otherwise there is no way to detect whether a single string is a list of values or a string. This is required because of the following yaml syntax.

Let's say a field takes a set of values:
```yaml
field:
     - value1
     - value2
```
When user needs to only pass a single value, they expect that the following syntax would also work.
```yaml
field: value1
```
To handle such situation we need to pass that information encoded in the string passed to bash so that from the conf.py. That is why this basic serializer encloses the comma seperated string with square brackets. Now since that string will also be need to deserialized, we'd need to enclose it ourselves when manually setting value for any variable such as in this case.